schema/auction_result.js

```diff
@@ -1,15 +1,26 @@
 import date from "./fields/date"
 import { amount } from "./fields/money"
 import { IDFields, NodeInterface } from "./object_identification"
-import { GraphQLFloat, GraphQLNonNull, GraphQLString, GraphQLObjectType } from "graphql"
+import { GraphQLFloat, GraphQLNonNull, GraphQLString, GraphQLObjectType, GraphQLEnumType } from "graphql"
 import { connectionDefinitions } from "graphql-relay"
 import { has } from "lodash"
 import Image from "schema/image"
 
+export const AuctionResultSorts = {
+  type: new GraphQLEnumType({
+    name: "AuctionResultSorts",
+    values: {
+      PRICE_AND_DATE_DESC: {
+        value: "-price_realized_cents_usd,-sale_date",
+      },
+    },
+  }),
+}
+
 const AuctionResultType = new GraphQLObjectType({
   name: "AuctionResult",
   interfaces: [NodeInterface],
-  isTypeOf: obj => has(obj, "dimension_text") && has(obj, "organization"),
+  isTypeOf: obj => has(obj, "sale_date") && has(obj, "organization"),
```

Comment:
@joeyAghion I noticed `dimension_text` was missing from some auction results, which causes a crash here. Could you help me figure out which unique fields will definitely be included? I wasn't able to tell from Diffusion. For now this seems to work with my sample artists.

Reply:
This kind of thing is definitely not guaranteed. In fact, `Lot`s in Diffusion can be very sparse (and so are light on `not null` fields) especially because some records have been sourced from 3rd parties or publications (e.g., so we may not even know _which_ auction house sold the lot, or the sale date).

Reply:
Sorry, meant to link to the `lots` schema:

https://github.com/artsy/diffusion/blob/2061d1a3c5d1a600cd7b350cea2af85b26031a8b/db/schema.rb#L24-L65

Reply:
It's totally fine that the field is `null`, but the field does need to be in the response.

How you can tell which fields wouldn't make it? Is there some general middleware/formatting in Diffusion that will strip out _all_ `null` fields (from the presenter, or something)?